### PR TITLE
Edit library list titles and posters

### PR DESCRIPTION
**Pull request by dulleh**
Probably should have someone else test these too ^^
Before:
image
After:
image
cc @JoshFabian

### CODE REVIEW

**Comment by matthewdias** [comment]
checkboxes are still off center?

**Comment by dulleh** [comment]
@matthewdias Checkboxes are off because of the 0.75rem padding on .table .td as it isn't applied to them. I'm not sure how to fix that without breaking the other columns.

**Comment by matthewdias** [comment]
@dulleh change https://github.com/hummingbird-me/hummingbird-client/blob/the-future/app/styles/pages/_library.scss#L471 to read `padding: 0.75rem 8px;` and then adjust the `line-height` until it looks right.

**Comment by dulleh** [comment]
image
@matthewdias works 👌

File: app/styles/components/_ratings.scss [hunk 1]
```diff
@@ -198,11 +198,6 @@
   }
 }
 
-// Hover element (rating-button)
-.rating-button-dropdown {
-  margin-bottom: 10px;
-}
-
```
Comment (trmcnvn) [inline app/styles/components/_ratings.scss]: This change breaks the rating widget on other parts of the site. This needs to be reversed and instead remove the margin on the library page only.

**Review by trmcnvn — CHANGES_REQUESTED** [review]

**Comment by dulleh** [comment]
I've just reverted for now because I don't know how to fix it without breaking other things (and don't have a local env. to test).

**Review by trmcnvn — APPROVED** [review]

**Pull request merged**